src/audio/rtnr/rtnr.c

```diff
@@ -36,7 +36,6 @@
 
```

Comment:
preferred format for subject is:

rtnr: Add audio stream layer...

instead of 

[RTNR] Add ...

Reply:
Fixed this. Thank you!

Reply:
@cujomalainey Are we able to merge this PR? Thanks!

Reply:
@cujomalainey Can this PR be merged? Thanks.